From cb132e96b7289945c17e2ac05481d0a551b747a8 Mon Sep 17 00:00:00 2001
From: Huang Shijie <b32955@freescale.com>
Date: Mon, 30 Aug 2010 09:56:06 +0800
Subject: [PATCH] ENGR00126721-3 MX23\MX28\MX50: Built-in the OCOTP driver in update_defconfig

  Turn on the OCOTP driver for the update def_config(imx23/28/50).

Signed-off-by: Huang Shijie <b32955@freescale.com>
---
 arch/arm/configs/imx23evk_updater_defconfig |    1 +
 arch/arm/configs/imx28evk_updater_defconfig |    1 +
 arch/arm/configs/imx5_updater_defconfig     |    1 +
 3 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/imx23evk_updater_defconfig b/arch/arm/configs/imx23evk_updater_defconfig
index 739e7bd..f6a430b 100644
--- a/arch/arm/configs/imx23evk_updater_defconfig
+++ b/arch/arm/configs/imx23evk_updater_defconfig
@@ -558,6 +558,7 @@ CONFIG_HW_RANDOM=y
 # CONFIG_I2C is not set
 # CONFIG_I2C_SLAVE is not set
 # CONFIG_SPI is not set
+CONFIG_FSL_OTP=y
 CONFIG_ARCH_REQUIRE_GPIOLIB=y
 CONFIG_GPIOLIB=y
 # CONFIG_GPIO_SYSFS is not set
diff --git a/arch/arm/configs/imx28evk_updater_defconfig b/arch/arm/configs/imx28evk_updater_defconfig
index 21223ed..fe9a908 100644
--- a/arch/arm/configs/imx28evk_updater_defconfig
+++ b/arch/arm/configs/imx28evk_updater_defconfig
@@ -531,6 +531,7 @@ CONFIG_SERIAL_CORE_CONSOLE=y
 # CONFIG_I2C is not set
 # CONFIG_I2C_SLAVE is not set
 # CONFIG_SPI is not set
+CONFIG_FSL_OTP=y
 CONFIG_ARCH_REQUIRE_GPIOLIB=y
 CONFIG_GPIOLIB=y
 # CONFIG_GPIO_SYSFS is not set
diff --git a/arch/arm/configs/imx5_updater_defconfig b/arch/arm/configs/imx5_updater_defconfig
index 4a06eb3..f594185 100644
--- a/arch/arm/configs/imx5_updater_defconfig
+++ b/arch/arm/configs/imx5_updater_defconfig
@@ -636,6 +636,7 @@ CONFIG_LEGACY_PTYS=y
 CONFIG_LEGACY_PTY_COUNT=256
 # CONFIG_IPMI_HANDLER is not set
 CONFIG_HW_RANDOM=y
+CONFIG_FSL_OTP=y
 # CONFIG_HW_RANDOM_TIMERIOMEM is not set
 # CONFIG_R3964 is not set
 # CONFIG_RAW_DRIVER is not set
-- 
1.5.4.4

